docs/Sonarr/V3/Sonarr-Release-Profile-RegEx-Anime.md

```diff
@@ -215,16 +213,17 @@ The reason most of these are added is due to their shitty quality or just in gen
 /(CuaP|PnPSubs|ICEBLUE|SLAX|U3-Web)/i,
 /(Raws-Maji|\bKRP\b|M@nI|Kanjouteki|PuyaSubs)/i,
 /\b(Beatrice|ohys|Kawaiika|neko|daddy)[ ._-]?(raws)\b/i,
-/\b(LowPower|Scryous)[ ._-]?(raws)\b/i,
+/\b(LowPower|Scryous|Skymoon)[ ._-]?(raws)\b/i,
 /\b(NS|AREY|BDMV|BDVD|BJX|DKB|DP|TnF)\b/i,
-/(Amb3r|DsunS|ExREN|\$tore-Chill)/i,
+/(Amb3r|DsunS|ExREN|\$tore-Chill|\bVF\b)/i,
```

Comment:
Nit: Group this with VOSFR for housekeeping purposes?

Reply:
I will reorganize this to more formatted soon but at the moment I try to keep them on one line for Sonarr readability. And that line is full at the moment. It is on my to-do list though to reorder things and have them placed together.